backend/src/test/resources/org-settings-query

```diff
@@ -11,6 +11,7 @@
       state
       zipCode
       phone
+      email
```

Comment:
This query is used in a test, which is great to see updated, only the actual test does not seem to be updated. That seems worth doing.

Reply:
After doing some digging, it looks like `org-settings-query` is not used in any of the existing tests... I am not super familiar with doing GraphQL tests, but I will try to add a test that uses it

Reply:
Oh, looks like that test was deleted in 07b881049dc83ef45c87a5ba18c6c5fd5a52f4d2. I am pretty sure it was not meant to stay deleted.

Reply:
Aha, should I add it back in from that commit then?

Reply:
That would be great!

I think the reason it got deleted was that there was a breaking schema change, so the JSON-path queries are probably wrong in annoying but fixable ways. As I'm about to start mucking with facilities from the other direction (devices and swabs) I personally would be very excited to have that regression test up and running again.

Reply:
Great, will do!